format.swift

```diff
@@ -0,0 +1,548 @@
+#!/usr/bin/env swift
+
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+#if canImport(Darwin)
+import Darwin.C
+#elseif canImport(Glibc)
+import Glibc
+#endif
+
+// MARK: - Logging
+
+// It’s a little unfortunate that we need to duplicate all these functions from
+// swift-syntax-dev-utils. But executing format.swift as a script is significantly
+// faster than running it from a package using `swift run` (~1s vs ~15s), which
+// makes a big difference when running the script multiple times.
```

Comment:
Little unfortunate indeed :(

Reply:
I can’t find the exact PR but @neonichu made some changes to improve `swift run` performance drastically by not having to re-build the build graph. With that change, we can make `format.py` part of swift-syntax-dev-utils`. 🎉